### ISSUE TITLE
Unable to load libxcrun

### ISSUE THREAD

**Issue by jonbstrong** [wrjlewis/notion-search-alfred-workflow#42]
The Notion search command has stopped working all of a sudden, not quite sure why..

Here's what the log output tells me:

[19:06:17.051] Notion Search[[Script Filter](alfredpreferences:workflows%3Eworkflow%3Euser.workflow.02B18B15-E79C-4773-BCF4-1E07E8720095%3E7DD3BDE5-A157-42E5-9376-F681FB50A4EE)] Queuing argument ''
[19:06:17.118] Notion Search[[Script Filter](alfredpreferences:workflows%3Eworkflow%3Euser.workflow.02B18B15-E79C-4773-BCF4-1E07E8720095%3E7DD3BDE5-A157-42E5-9376-F681FB50A4EE)] Script with argv '(null)' finished
[19:06:17.120] ERROR: Notion Search[[Script Filter](alfredpreferences:workflows%3Eworkflow%3Euser.workflow.02B18B15-E79C-4773-BCF4-1E07E8720095%3E7DD3BDE5-A157-42E5-9376-F681FB50A4EE)] Code 1: xcrun: error: unable to load libxcrun (dlopen(/Library/Developer/CommandLineTools/usr/lib/libxcrun.dylib, 0x0005): tried: '/Library/Developer/CommandLineTools/usr/lib/libxcrun.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e')), '/usr/lib/libxcrun.dylib' (no such file)).

**Comment by wrjlewis** [comment]
Are you on a M1 Mac, and recently updated perhaps? 

I'm not sure but after a quick google search this appears to be when command line tools have become out of date. This workflow runs on python3 which is included in command line tools, hence the location of the error. 

You could try reinstalling mac command line tools, as seen [here](https://stackoverflow.com/questions/71298816/truffle-failing-to-install-on-a-mac-m1)

Basically...

junk the existing folder

`sudo rm -rf /Library/Developer/CommandLineTools
re-install`

Reinstall

`sudo Xcode-select --install `

**Comment by jonbstrong** [comment]
After re-installing command line tools, it works!
Thanks :)

**Comment by wrjlewis** [comment]
Awesome
